### PR TITLE
 test caption edit

### PR DESCRIPTION
**Pull request by aslater3**
Test pull - added caption specific for andy
Closes #2

### CODE REVIEW

**Comment by aslater3** [comment]
Fixes #2

**Comment by aslater3** [comment]
References #2

**Comment by aslater3** [comment]
Closes #2

File: _slides/05-aslater3.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Andy Edited this
```
Comment (e-mrazek) [inline _slides/05-aslater3.md:6]: Looks good

**Review by e-mrazek — APPROVED** [review]

**Pull request merged**